src/motion_capture.cc

```diff
@@ -131,8 +131,8 @@ void MotionCapturePlugin::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
   // Create our ROS node. This acts in a similar manner to
   // the Gazebo node
   this->ros_node_.reset(new ros::NodeHandle("gazebo_client"));
-  std::string topic_name = "/human_joint_pos";
-  this->publisher_ = this->ros_node_->advertise<custom_robot_msgs::PositionsHeadered>(topic_name, 1000);
+  std::string topic_name = "/human_pose_measurement";
```

Comment:
I would suggest to use a parent topic to better cluster the messages, e.g., `/sara-shield/human_pose_measurement`.
What do you think?

Reply:
Makes sense. Fixed in https://github.com/manuelvogel12/human-gazebo/commit/c4d1592bb77396c603ac5d947884912fb843b1b5
and followup commit https://github.com/manuelvogel12/human-gazebo/commit/c4d1592bb77396c603ac5d947884912fb843b1b5, since topics cannot contain "-",  the topic is now `/sara_shield/human_pose_measurement`